packages/common/components/ads/gam-sticky-leaderboard.marko

```diff
@@ -0,0 +1,13 @@
+import getAdUnit from '@endeavorb2b/base-website-common/utils/gam/get-adunit';
+import { getAsArray } from '@base-cms/object-path';
+
+$ const { site } = out.global;
+$ const { name } = input;
+$ const aliases = getAsArray(input, 'aliases');
+$ const adunit = getAdUnit(site, { name, aliases })
+$ const { path, sizeMapping, size: sizes } = adunit;
+$ const props = { path, sizeMapping, sizes, ...input };
```

Comment:
Why is the input being re-applied? This could potentially overwrite the `path`, `sizeMapping` and `size` that's being created by `getAdUnit`